calculator.go

```diff
@@ -1,13 +1,86 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
+import (
+	"errors"
+	"fmt"
+	"go/token"
+	"go/types"
+	"math"
+	"strconv"
+)
+
 // Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+func Add(n1, n2 float64, extra ...float64) float64 {
+	var total float64 = n1 + n2
+	for i := range extra {
+		total += extra[i]
+	}
+	return total
 }
 
 // Subtract takes two numbers and returns the result of subtracting the second
 // from the first.
-func Subtract(a, b float64) float64 {
-	return b - a
+func Subtract(n1, n2 float64, extra []float64) float64 {
```

Comment:
I think `a, b` is a better choice here; they're just easier to tell apart, visually, than `n1, n2`.